src/main/java/org/openmaptiles/layers/Park.java

```diff
@@ -91,20 +91,15 @@ public Park(Translations translations, PlanetilerConfig config, Stats stats) {
 
```

Comment:
is this handled?

https://github.com/openmaptiles/openmaptiles/compare/e61442c49dd32974d4952046b4149202c7072d59...master#diff-2188c847bfd73aefebe7c11baa9089f8f09b34160d8a2e2debb1a0d58b707e15